src/main/java/org/jabref/gui/entryeditor/citationrelationtab/semanticscholar/SemanticScholarFetcher.java

```diff
@@ -16,7 +15,7 @@ public class SemanticScholarFetcher implements CitationFetcher {
     private static final String SEMANTIC_SCHOLAR_API = "https://api.semanticscholar.org/graph/v1/";
 
     @Override
-    public List<BibEntry> searchCitedBy(BibEntry entry) throws FetcherException {
+    public List<BibEntry> searchCitedBy(BibEntry entry) {
```

Comment:
Please keep the exception - AND remove "throw new RuntimeException"

Reply:
I can't keep `throws FetcherException` because the method doesn't throw it.